backend/routes/api/quickstarts/quickStartUtils.js

```diff
@@ -19,19 +22,21 @@ const getInstalledQuickStarts = async (fastify) => {
     installedQuickStarts.push(...res.body.items);
   } catch (e) {
     fastify.log.error(e, 'failed to get quickstarts');
-    const error = createError(500, 'failed to get quickstarts');
-    error.explicitInternalServerError = true;
-    error.error = 'failed to get quickstarts';
-    error.message =
-      'Unable to load Kubeflow resources. Please ensure the Open Data Hub operator has been installed.';
-    throw error;
+    if (!constants.USE_MOCK_DATA || USE_LOCAL_QUICK_STARTS) {
```

Comment:
review: this condition gives the impression it will always run based on USE_LOCAL_QUICK_STARTS being set to true on line 6, just seems like a vague order diff with line 35

Reply:
Again, temporary code.